python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat.py

```diff
@@ -164,15 +164,15 @@ async def collect_output_messages(
     async def run(
         self,
         *,
-        task: str | ChatMessage | None = None,
+        task: str | ChatMessage | list[ChatMessage] | None = None,
```

Comment:
use `List[ChatMessage]`

Reply:
We need to update the protocol `TaskRunner` and corresponding implementations.

Reply:
- Updated type hints to use `List` from `typing` module instead of built-in `list` in both `_task.py` and `_base_group_chat.py`
- Added proper import statements: `from typing import List` where needed